dev/tasks/tasks.yml

```
@@ -1582,6 +1582,10 @@ tasks:
         # ensure we have at least one build with parquet encryption disabled
         PARQUET_REQUIRE_ENCRYPTION: "OFF"
       {% endif %}
+      {% if pandas_version == "nightly" %}
+        # TODO can be removed once this is enabled by default in pandas >= 3
```

I was confused on where we were using this and realized is a Pandas thing, maybe we can point it on the comment?
```suggestion
        # TODO can be removed once this is enabled by default in pandas >= 3
        # This is to enable the Pandas feature.
        # See: https://github.com/pandas-dev/pandas/pull/58459